### DISCUSSION TITLE
Double start_wallet.exe process stall themselves on 2 keys in the same GUI Windows

### DISCUSSION THREAD

**Discussion by giovanniPepi** [Chia-Network/chia-blockchain#6926] [Ideas]
I'm sorry if this is already being addressed by the developer team, it's my first time posting something on GitHub

When there is 2 keys in the same client/GUI in windows, the status on the Wallet tab keeps switching from syncing to not synced. Upon further investigation, I found out 2 start_wallet.exe processes running (there is usually only one); and upon terminating one of them through task manager, the desired wallet finally synced.

![image](https://user-images.githubusercontent.com/5255535/123019949-e03c9d00-d3a7-11eb-90af-4a46e2cd58b8.png)

There could be an option to either sync only one wallet, or to have them running at the same time without crashing the other

**Comment by loppefaaret** [comment]
you _should_ only have a single wallet process running - if it somehow got an extra opened up it would exhibit the behavior you explained - it is currently only possible to have 1 wallet active - it is planned in the future to enable multiple wallets being active at the same time.
